release-notes/1.0/known-issues-1.0.0.md

```diff
@@ -0,0 +1,232 @@
+# 1.0.0 Known Issues
+
+This document lists known issues for *.NET Core 1.0.0* and *.NET Core SDK 1.0.0 Preview 2* which may be encountered during usage.
+
+## NegotiateStream
+
+```NegotiateStream``` relies on the gssapi implementation available on the platform.
+* On Linux flavors, the default gssapi implementation provided is [MIT's krb5 library](http://web.mit.edu/kerberos/) which is available on all the linux platforms
+* On OSX, the default implementation is heimdal-based GSS.framework
+
+On both Linux and OSX, NegotiateStream uses SPNEGO and relies on the underlying implementation for supporting Kerberos or NTLM as the underlying security protocol.
+
+* On OSX, Gss.framework supports SPNEGO mechanism with Kerberos and NTLM as the security protocols.
+* On Linux, mit krb5 library supports SPENGO mechanism and Kerberos as the out of the box security protocol. The implenentation can be made to support NTLM too by installing [GSS-NTLMSSP plugin](https://fedorahosted.org/gss-ntlmssp/) or another plugin with similar functionality. The fallback to NTLM is thus dependent on runtime availability of such a plugin.
+
+On RHEL and CENTOS, the plugin GSS-NTLMSSP is available from the package managers. The same is also available on Ubuntu 16. It can be installed on the other flavors of linux.
+
+There's no compile time dependency on this plugin.
+
+## Windows 7
+
+**Issue:**
```

Comment:
Can we point this to the pre-req document @blackdwarf wrote so that folks can install the required dependencies (which is what these "issues" are about)?